src/extension/background-script/actions/nostr/decryptOrPrompt.ts

```diff
@@ -33,9 +33,13 @@ const decryptOrPrompt = async (message: MessageDecryptGet, sender: Sender) => {
         rememberPermission: boolean;
       }>({
         ...message,
-        action: "public/nostr/confirm",
+        action: "public/nostr/confirmEncryptOrDecrypt",
         args: {
-          description: i18n.t("permissions:nostr.nip04decrypt"),
+          description: {
+            action: i18n.t("permissions:nostr.nip04decrypt"),
+            peer: message.args.peer,
+            message: message.args.ciphertext,
```

Comment:
@reneaaron what do you think. 
This ciphertext will not be useful for the user. so I'd say we should leave this out here. What do you think?

Reply:
I agree, but then we'd need to have an alternative layout here. I'd suggest creating a follow-up task to split encrypt / decrypt into two different flows. I'd rather have them separate...

Reply:
Created an issue cc @pavanjoshi914 #2942